<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreatNewProject</class>
 <widget class="QDialog" name="CreatNewProject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>470</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>470</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>470</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>创建新项目</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog {
    background-color: white;
}


QLabel {
    color: black;
    background-color: transparent;
    border: none;
}

QLabel#titleLabel {
    font: 20px 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';
    padding: 0;
}

QLabel#windowTitleLabel {
    font: 12px 'Segoe UI', 'Microsoft YaHei', 'PingFang SC';
    padding: 6px 6px;
    background-color: rgb(243, 243, 243);
}

</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>9</y>
     <width>201</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(147, 147, 147);

font: 18pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>创建新项目</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projectName">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>420</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>420</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(147, 147, 147);

font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>项目名称:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(147, 147, 147);

font: 12pt &quot;Microsoft YaHei UI&quot;;</string>
   </property>
   <property name="text">
    <string>保存位置:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>420</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: black;
    background: rgba(255, 255, 255, 0.7);
    border: 1px solid rgba(0, 0, 0, 0.073);
    border-bottom: 1px solid rgba(0, 0, 0, 0.183);
    border-radius: 5px;
	font: 15pt &quot;Microsoft YaHei UI&quot;;
    padding: 5px 12px 6px 12px;
    outline: none;
}


QPushButton[isPlaceholderText=true] {
    color: rgba(0, 0, 0, 0.6063);
}

QPushButton[hasIcon=false] {
    padding: 5px 12px 6px 12px;
}

QPushButton[hasIcon=true] {
    padding: 5px 12px 6px 36px;
}


QPushButton:hover {
    background: rgba(249, 249, 249, 0.5);
}

QPushButton:pressed{
    color: rgba(0, 0, 0, 0.63);
    background: rgba(249, 249, 249, 0.3);
    border-bottom: 1px solid rgba(0, 0, 0, 0.073);
}

QPushButton:disabled {
    color: rgba(0, 0, 0, 0.36);
    background: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 0.06);
    border-bottom: 1px solid rgba(0, 0, 0, 0.06);
}


</string>
   </property>
   <property name="text">
    <string>创建</string>
   </property>
  </widget>
  <widget class="QListWidget" name="templateList">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>41</y>
     <width>451</width>
     <height>371</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QListWidget::item 
{ 
	min-height: 90px;
	 max-height: 120px; 

}</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="projectPath">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>460</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>C:/ProgramData</string>
   </property>
  </widget>
  <widget class="QPushButton" name="selectPath">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>460</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>600</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    color: black;
    background: rgba(255, 255, 255, 0.7);
    border: 1px solid rgba(0, 0, 0, 0.073);
    border-bottom: 1px solid rgba(0, 0, 0, 0.183);
    border-radius: 5px;
	font: 10pt &quot;Microsoft YaHei UI&quot;;
    padding: 5px 12px 6px 12px;
    outline: none;
}


QPushButton[isPlaceholderText=true] {
    color: rgba(0, 0, 0, 0.6063);
}

QPushButton[hasIcon=false] {
    padding: 5px 12px 6px 12px;
}

QPushButton[hasIcon=true] {
    padding: 5px 12px 6px 36px;
}


QPushButton:hover {
    background: rgba(249, 249, 249, 0.5);
}

QPushButton:pressed{
    color: rgba(0, 0, 0, 0.63);
    background: rgba(249, 249, 249, 0.3);
    border-bottom: 1px solid rgba(0, 0, 0, 0.073);
}

QPushButton:disabled {
    color: rgba(0, 0, 0, 0.36);
    background: rgba(249, 249, 249, 0.3);
    border: 1px solid rgba(0, 0, 0, 0.06);
    border-bottom: 1px solid rgba(0, 0, 0, 0.06);
}


</string>
   </property>
   <property name="text">
    <string>...</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
